### PR TITLE
in RemoveFilesCommand request was replaced with publish

### PR DESCRIPTION
**Pull request by vorobevamv**

### CODE REVIEW

File: src/ProjectService.Business/Commands/File/RemoveFilesCommand.cs [hunk 1]
```diff
@@ -93,16 +48,13 @@ public async Task<OperationResultResponse<bool>> ExecuteAsync(RemoveFilesRequest
...
 
-      bool result = await RemoveFilesAsync(request.FilesIds, response.Errors);
+      response.Body = await _repository.RemoveAsync(request.FilesIds);
 
-      if (!result)
+      if (response.Body)
       {
-        return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.BadRequest, response.Errors);
+        await _publish.RemoveFilesAsync(request.FilesIds);
       }
```
Comment (MariyaSin) [inline src/ProjectService.Business/Commands/File/RemoveFilesCommand.cs:56]: add else and set response = responseCreator......

Comment (vorobevamv) [reply src/ProjectService.Business/Commands/File/RemoveFilesCommand.cs:54]: added

File: src/ProjectService.Validation/Project/EditProjectRequestValidator.cs [hunk 2]
```diff
@@ -87,7 +87,7 @@ private async Task HandleInternalPropertyValidationAsync(Operation<EditProjectRe
         x => x == OperationType.Replace,
         new Dictionary<Func<Operation<EditProjectRequest>, bool>, string>
         {
-          { x => x.value == null || x.value.ToString().Trim().Length < 30, "Short name is too long." },
+          { x => x.value == null || x.value.ToString().Trim().Length < 40, "Short name is too long." },
```
Comment (MariyaSin) [inline src/ProjectService.Validation/Project/EditProjectRequestValidator.cs:90]: 41))

**Review by ndoriomedov — APPROVED** [review]

File: Common.Configurations.targets [hunk 3]
```diff
@@ -26,9 +26,9 @@
   </PropertyGroup>
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
-      <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.61" />
+      <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.62" />
```
Comment (MariyaSin) [inline Common.Configurations.targets:29]: update to 63 and update the service specification

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**